src/main/java/goodgenerator/crossmod/LoadedList.java

```diff
@@ -1,26 +1,30 @@
 package goodgenerator.crossmod;
 
-import cpw.mods.fml.common.Loader;
+import static gregtech.api.enums.Mods.Automagy;
+import static gregtech.api.enums.Mods.GTPlusPlus;
+import static gregtech.api.enums.Mods.NewHorizonsCoreMod;
+import static gregtech.api.enums.Mods.ThaumicBases;
+import static gregtech.api.enums.Mods.ThaumicEnergistics;
+import static gregtech.api.enums.Mods.ThaumicTinkerer;
+import static gregtech.api.enums.Mods.WitchingGadgets;
 
 public class LoadedList {
 
     public static boolean GTPP;
     public static boolean GTNH_CORE;
-    public static boolean BOTDUSTRIES;
     public static boolean THAUMIC_BASES;
     public static boolean THAUMIC_TINKERER;
     public static boolean AUTOMAGY;
     public static boolean WITCHING_GADGETS;
     public static boolean THAUMIC_ENERGISTICS;
 
     public static void init() {
-        GTPP = Loader.isModLoaded("miscutils");
-        GTNH_CORE = Loader.isModLoaded("dreamcraft");
-        BOTDUSTRIES = Loader.isModLoaded("botdustries");
-        THAUMIC_BASES = Loader.isModLoaded("thaumicbases");
-        THAUMIC_TINKERER = Loader.isModLoaded("ThaumicTinkerer");
-        AUTOMAGY = Loader.isModLoaded("Automagy");
-        WITCHING_GADGETS = Loader.isModLoaded("WitchingGadgets");
-        THAUMIC_ENERGISTICS = Loader.isModLoaded("thaumicenergistics");
+        GTPP = GTPlusPlus.isModLoaded();
+        GTNH_CORE = NewHorizonsCoreMod.isModLoaded();
```

Comment:
there is no need for those global variables, imo they should be replaced by the call of the Enum directly.